terragrunt/Dockerfile

```diff
@@ -1,10 +1,9 @@
 # Container to run Terragrunt and Terraform
-# Contains inside TFenv and TGSwitch to allow users to install custom Terraform and Terragrunt versions
+# Contains inside mise to allow users to install custom Terraform and Terragrunt versions
 FROM ubuntu:22.04
-MAINTAINER Gruntwork <info@gruntwork.io>
+LABEL maintainer "Gruntwork <info@gruntwork.io>""
```

Comment:
ERROR: failed to solve: failed to process "\"Gruntwork <info@gruntwork.io>\"\"": unexpected end of statement while looking for matching double-quote